lib/core/viewmodels/more_viewmodel.dart

```diff
@@ -0,0 +1,156 @@
+  String get appVersion => _appVersion;
+
+  @override
+  Future futureToRun() async {
+    setBusy(true);
+    await PackageInfo.fromPlatform()
+        .then((value) => _appVersion = value.version);
+    setBusy(false);
+    return true;
+  }
+
+  /// Used to logout user, delete cache, and return to login
+  Future<void> logout(BuildContext context) async {
+    await _cacheManager.empty();
+    UserRepository().logOut();
+    // Dismiss alertDialog
+    Navigator.of(context).pop();
+    _navigationService.pushNamedAndRemoveUntil(RouterPaths.login);
+    showToast(AppIntl.of(context).login_msg_logout_success);
+  }
+
+  /// Used to send feedback
+  Future<void> sendFeedback(BuildContext context, String feedbackText,
+      Uint8List feedbackScreenshot) async {
+    final File file = await _localFile;
+    await file.writeAsBytes(image.encodePng(
+        image.copyResize(image.decodeImage(feedbackScreenshot), width: 307)));
+
+    /// Create a GitHub Client
+    const String githubApiToken = String.fromEnvironment('GITHUB_API_TOKEN');
+    final github = GitHub(auth: Authentication.withToken(githubApiToken));
+
+    _uploadFileToGithub(github, file);
+
+    await _createGithubIssue(github, file, feedbackText);
+
+    file.deleteSync();
+    showToast(
+      AppIntl.current?.thank_you_for_the_feedback,
+      position: ToastPosition.center,
+    );
+    BetterFeedback.of(context).hide();
+  }
+
+  /// License text box
+  List<Widget> aboutBoxChildren(TextStyle textStyle) {
+    return <Widget>[
+      const SizedBox(height: 24),
+      RichText(
+        text: TextSpan(
+          children: <TextSpan>[
+            TextSpan(style: textStyle, text: AppIntl.current.flutter_license),
+            TextSpan(
+                style: textStyle.copyWith(color: Colors.blue),
+                text: AppIntl.current.flutter_website,
+                recognizer: TapGestureRecognizer()
+                  ..onTap =
+                      () => Util().launchURL(AppIntl.current.flutter_website)),
+            TextSpan(style: textStyle, text: '.'),
+          ],
+        ),
+      ),
+    ];
+  }
+
+  /// Create Github issue
+  Future<void> _createGithubIssue(
+      GitHub github, File file, String feedbackText) async {
+    final PackageInfo packageInfo = await PackageInfo.fromPlatform();
+    github.issues.create(
+        RepositorySlug.full("ApplETS/Notre-Dame"),
+        IssueRequest(
+            title: 'Issue from ${packageInfo.appName} ',
+            body: " **Describe the issue** \n"
+                "```$feedbackText```\n\n"
+                "**Screenshot** \n"
+                "![screenshot](https://github.com/ApplETS/Notre-Dame-Bug-report/blob/main/${file.path.replaceFirst('storage/emulated/0/Android/data/ca.etsmtl.applets.notredame/files/', '')}?raw=true)\n\n"
+                "**Device Infos** \n"
+                "- **App name:** ${packageInfo.appName} \n"
+                "- **Package name:** ${packageInfo.packageName}  \n"
```

Comment:
Do you think this is really useful? After all the App name is ÉTSMobile and will stay like this, the package name will not change either ^^'

Reply:
true